src/item.cpp

```diff
@@ -7381,7 +7381,7 @@ void item::process_temperature_rot( float insulation, const tripoint &pos,
         return;
     }
 
-    int temp = g->weather.get_temperature( pos );
+    int temp = std::max( g->weather.get_temperature( pos ), -459 );
```

Comment:
Make `-459` a constant?

Reply:
It is now constant `temperatures::lowest_temperature `